public/src/style.scss

```diff
@@ -146,15 +145,15 @@
 
 #content {
     transition: margin-left ease-in-out 0.2s;
-    margin-left: 350px;
-    margin-right: 100px;
+    margin-left: 0px;
```

Comment:
This class (#content) and #content-full are used to adjust the main content based on whether the side bar is open or not.
Removing these causes all other components to look funky.